app/models/user.rb

```diff
@@ -1,3 +1,8 @@
 class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
```

Comment:
- [ ] Good job so far 👍 A user should also have many likes. Kindly add that too 👍